client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -37,6 +37,28 @@
 
 public class RssSparkConfig {
 
+  public static final ConfigOption<Boolean> RSS_RESUBMIT_STAGE_ENABLED =
```

Comment:
I see all the configs start with `RSS_RESUBMIT_STAGE_`, but the keys of them all start with `rss.stageRetry`.

Can we unify these two terms, "stage resubmit" and "stage retry"? We should use only one of them in our project.

Reply:
Emm... Yes. but this could be optimized in the final PR. I prefer `stage retry`

Reply:
OK.